api/src/main/resources/client_api.yaml

```diff
@@ -177,7 +190,61 @@ definitions:
         items:
           type: string
 
+  ColumnFilter:
+    type: object
+    required:
+      - columnName
+    description: >
+      A filter applied to the results of a query, based on a column value in a table. Only results
+      matching the filter will be returned. One (and only one) of the value columns should be populated.
+      values and valueNumbers should only be used in conjunction with the "IN" operator.
+    properties:
+      columnName:
+        description: >
+          The name of the column to filter on.
+        type: string
+      operator:
+        description: >
+          The operator to use when comparing values. Defaults to EQUAL. If the "IN" operator is used,
```

Comment:
nit: uppercase "in" in the Operator enum or lowercase "IN" in the docs throughout